src/main/java/com/gradle/enterprise/conventions/customvalueprovider/LocalBuildCustomValueProvider.java

```diff
@@ -29,7 +29,7 @@ public void accept(Settings settings, BuildScanExtension buildScan) {
             }
         }
 
-        GradleEnterpriseConventions.execAndGetStdout(settings.getRootDir(), "git", "log", "-1", "--format=%H")
+        GradleEnterpriseConventions.execAndGetStdout(settings.getRootDir(), "git", "log", "-1", "--format=%H", "--no-show-signature")
```

Comment:
Why don't we use `git rev-parse HEAD`? What is the downside of this command?

Reply:
No idea, I do not even understand the description of that command tbh - https://git-scm.com/docs/git-rev-parse
I can confirm it gives the same result in my current situation though.

Let me adapt the PR to use that instead and see if build it OK.

Reply:
Fixed, I even left the `log` comment in the test, so that if one day they diverge, the test will say it.